include/depthai/depthai_wrapper.hpp

```diff
@@ -0,0 +1,70 @@
+#include "depthai/device.hpp"
+#include "nlohmann/json.hpp"
+#include <opencv2/core/core.hpp>
+#include <opencv2/opencv.hpp>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace DepthAI {
+
+using CV_mat_ptr = std::shared_ptr<cv::Mat>; // shared ptr for cv::Mat
+using PacketsTuple = std::tuple<
+    std::list<std::shared_ptr<NNetPacket>>,
+    std::list<std::shared_ptr<HostDataPacket>>>; // tuple of data packets on which Depthai is
+// publishing the streams
```

Comment:
Please move comments above the lines to prevent them from going to the next line. Comments next to code are only for very short explanations or clarifications.